countryspec/client.go

```diff
@@ -0,0 +1,78 @@
+// Package countryspec provides the /country_specs APIs
+package countryspec
+
+import (
+	"net/url"
+
+	stripe "github.com/stripe/stripe-go"
+)
+
+// Client is used to invoke /country_specs and countryspec-related APIs.
+type Client struct {
+	B   stripe.Backend
+	Key string
+}
+
+// Get returns a CountrySpec for a given country code
+// For more details see https://stripe.com/docs/api/ruby#retrieve_country_spec
+func Get(country string) (*stripe.CountrySpec, error) {
+	return getC().Get(country)
+}
+
+func (c Client) Get(country string) (*stripe.CountrySpec, error) {
+	countrySpec := &stripe.CountrySpec{}
+	err := c.B.Call("GET", "/country_specs/"+country, c.Key, nil, nil, countrySpec)
+
+	return countrySpec, err
+}
+
+// List lists available CountrySpecs.
+func List(params *stripe.CountrySpecListParams) *Iter {
+	return getC().List(params)
+}
+
+func (c Client) List(params *stripe.CountrySpecListParams) *Iter {
+	type CountrySpecList struct {
```

Comment:
Fairly small, but at least a few of the other methods start with a lower case letter (i.e. `countrySpecList`). I don't think it matters here, but it normally conveys a "non-exported" name (i.e. internal to the package).